shell/shell.go

```diff
@@ -42,15 +42,17 @@ type wrapper struct {
 	ctx context.Context
 	cmd *exec.Cmd
 
-	path            string
-	dir             string
-	args            []string
-	env             []string
-	osEnv           bool
-	sysProcAttr     *syscall.SysProcAttr
+	path        string
+	dir         string
+	args        []string
+	env         []string
+	osEnv       bool
+	sysProcAttr *syscall.SysProcAttr
+
 	ctxCancellation bool
+	killedByCancel  uint32 // Number of times the command was "killed" after the context was canceled.
 
-	// When a context is provided and it is cancelled while the process is
+	// When a context is provided and it is canceled while the process is
```

Comment:
https://www.grammarly.com/blog/canceled-vs-cancelled/ 🤷‍♂️ Canadian English typically takes the UK spelling in this case but really either is completely fine.

Reply:
I changed it because that was `context.Canceled` uses.